manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml

```diff
@@ -118,7 +118,7 @@ spec:
     must be explicitly created by a cluster administrator (e.g. `oc create ns openshift-logging`). To enable metrics
     service discovery add namespace label `openshift.io/cluster-monitoring: "true"`.
 
-    For additional installation documentation see [Deploying cluster logging](https://docs.openshift.com/container-platform/4.1/logging/efk-logging-deploying.html)
+    For additional installation documentation see [Deploying cluster logging](https://docs.openshift.com/container-platform/4.5/logging/cluster-logging-deploying.html)
```

Comment:
@jcantrill As far as I know, there is not an OCP set that points to master. Master is not version-specific.  Master is our working branch is the complete set of docs for all versions of OCP.  
The reason for the 404 is that the 4.5 branch has not been created and, as I understand,  won't get created until 4.4 releases. 
@vikram-redhat Thoughts?

Reply:
You can use https://docs.openshift.com/container-platform/latest/logging/cluster-logging-deploying.html

Latest will resolve to whatever it is currently.

Reply:
A link pointing to the latest version of the docs will only be correct for the latest version of the operator. We've got 4.5 in the name of the CSV YAML, we should have 4.5 in the docs link. It will be live when the operator GAs.

Reply:
Ah ... in that case, we could replace it with {product-version} which will resolve to the version of the docs this is in, if I am understanding this right.

Reply:
This text is not in the docs, it's in this operator's tile on the OperatorHub UI, as in https://miro.medium.com/max/930/1*yYq_enSdlDTi2_qGQEnWGw.jpeg. 
4.5 is all over this YAML, although with comments # The version value is substituted by the ART pipeline. Don't know how that subst'ing works though.